maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt

```diff
@@ -130,6 +132,46 @@ class ApiClient(
         }
     }
 
+    fun render(
+        screenRecording: File,
+        frames: List<ResultView.Frame>,
+        progressListener: (totalBytes: Long, bytesWritten: Long) -> Unit = { _, _ -> },
+    ): String {
+        val baseUrl = "https://maestro-record.ngrok.io"
```

Comment:
I suggest we setup a redirect somewhere on our end to be able to move away from ngrok later on